src/app/admin/adminApp.js

```diff
@@ -56,6 +236,7 @@ export default function AdminPage() {
         list={ListGuesser}
         edit={EditGuesser}
         show={ShowGuesser}
+        create={SpecialistCreate}
```

Comment:
This attribute ```create={SpecialistCreate}``` is related to the specialist component, not the therapy. You should move it down to line 245.